jitbuilder/release/.gitignore

```diff
@@ -0,0 +1,33 @@
+###############################################################################
+#
+# (c) Copyright IBM Corp. 2016, 2016
+#
+#  This program and the accompanying materials are made available
+#  under the terms of the Eclipse Public License v1.0 and
+#  Apache License v2.0 which accompanies this distribution.
+#
+#      The Eclipse Public License is available at
+#      http://www.eclipse.org/legal/epl-v10.html
+#
+#      The Apache License v2.0 is available at
+#      http://www.opensource.org/licenses/apache2.0.php
+#
+# Contributors:
+#    Multiple authors (IBM Corp.) - initial implementation and documentation
+###############################################################################
+
+include/compiler/
```

Comment:
No. This file only ignores the generated headers and binaries, since there are a few tracked files in [jitbuilder/release](https://github.com/eclipse/omr/tree/master/jitbuilder/release). This file would only need to be changed if there is a new file that should not be tracked, most likely a new test binary.

It's possible to add jitbuilder/release to the top level .gitignore, however we would need to add 7 exceptions for the folders and files that are supposed to be tracked. This may also cause confusion where `git status` won't show any new files in this directory. Personally, I think this makes about as much sense as ignoring the top level directory of OMR and whitelisting the files and folders we want to track.